language/move-prover/tests/sources/aborts-if.exp

```diff
@@ -0,0 +1,40 @@
+returns: exiting with boogie verification errors
+error:  A postcondition might not hold on this return path.
+
+    ┌── tests/sources/aborts-if.move:16:9 ───
```

Comment:
Just for my edification, what is the point of the vertical bar on the left of the error message?  The number labeling the vertical bar seems to be identical to the line number attached to the file name below the error message.  Isn't that redundant?

Reply:
Not my design, comes from crate codespan and is used everywhere where Rust is in place (e.g. also by the move-lang compiler). But I like it.